Test.java

```diff
@@ -0,0 +1,12 @@
+class MyClass {
+  MyClass(MyClass mc) { }
+  
+  int     foo1() { return 0; }
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method and declare a constant for this value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS3400)